src/main/java/org/radarcns/redcap/config/RedCapInfo.java

```diff
@@ -135,6 +139,13 @@ public String getIntegrationForm() {
 
     public Set<Attribute> getAttributes() { return attributes; }
 
+    public List<String> getAttributeFieldNames() {
+        List<String> fieldNames = new ArrayList<>();
+        if(attributes == null || attributes.isEmpty())
+            return fieldNames;
```

Comment:
please add braces.
You can also just return `Collections.emptyList()` here, so you don't have to initialise and assign a separate variable.